package.json

```diff
@@ -72,7 +72,9 @@
     "es6-symbol": "*",
     "eslint-import-resolver-node": "^0.2.0",
     "lodash.cond": "^4.3.0",
+    "lodash.endswith": "^4.0.1",
     "object-assign": "^4.0.1",
     "pkg-up": "^1.0.0"
+    "object-assign": "^4.0.1"
```

Comment:
`object-assign` is duplicated (and linked to that: there seems to be missing a comma after `pkg-up`).